server/server.go

```diff
@@ -212,6 +212,12 @@ func NewServer(config *Config) (*Server, error) {
 			m.config.Chain.Params.ContractDeployerAllowList)
 	}
 
+	// apply allow list genesis data
```

Comment:
```suggestion
	// apply transactions execution allow list genesis data
```

Reply:
Similar would go for `ContractDeployerAllowList`, modify comment like this for example:
// apply allow list contracts deployer genesis data

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1349/commits/a9ee37f3e386d6b41b8bb11e956076c54e0c725a